pkgs/tools/misc/mutagen-compose/default.nix

```diff
@@ -1,28 +1,37 @@
-{ stdenv, lib, buildGoModule, fetchFromGitHub, fetchzip }:
+{
+  stdenv,
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  fetchzip,
+  # Used to enable SSPL licensed code.
+  withSSPL ? false,
+}:
 
 buildGoModule rec {
   pname = "mutagen-compose";
-  version = "0.16.5";
+  version = "0.17.0";
 
   src = fetchFromGitHub {
     owner = "mutagen-io";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-Rn3aXwez/WUGpuRvA6lkuECchpYek8KDMh6xzZOV9v0=";
+    sha256 = "sha256-kw+/HG6UfbocWeCoI2Bk/wjL+4lUl7B/OOg/ImYTOQQ=";
   };
 
-  vendorHash = "sha256-EkLeB2zUJkKCWsJxMiYHSDgr0/8X24MT0Jp0nuYebds=";
+  vendorHash = "sha256-A4nLRzgkg8xbODlKyNRg04Qpwd8tUyG44M7zYw1p96U=";
 
   doCheck = false;
 
   subPackages = [ "cmd/mutagen-compose" ];
 
+  tags = [ "mutagencompose" ] ++ lib.optional withSSPL "mutagensspl";
```

Comment:
I think you need to include `mutagencli` as well in order for the `mutagen` cli to work as well per https://github.com/mutagen-io/mutagen/blob/master/cmd/mutagen/tagcheck.go#LL1C13-L1C23 & local testing :D

Reply:
Oh I can't read, I read this as for `mutagen` itself. Please ignore!